app/views/users/show.html.slim

```diff
@@ -28,3 +28,8 @@
         br
         i class="fa fa-phone"
         = @user.phone
+- if user_signed_in? && current_user.id == params[:id].to_i
+  javascript:
```

Comment:
condition for marking a profile link only when it's visited by current user (can't remove to_i here because of custom id)